config/jobs/kubernetes/sig-testing/make-test.yaml

```diff
@@ -14,6 +14,10 @@ presubmits:
     always_run: true
     path_alias: k8s.io/kubernetes
     spec:
+      # unit tests have no business requiring root or doing privileged operations
+      securityContext:
+        runAsUser: 2000
+        allowPrivilegeEscalation: false
```

Comment:
allowPrivilegeEscalation doesn't exist at the pod spec level... I think we have to drop that field

Reply:
Done